localization/imu_bias_tester/readme.md

```diff
@@ -0,0 +1,11 @@
+\page imubiastester Imu Bias Tester 
```

Comment:
Can you explain in which circumstances we use this? If not during operations or in the robot, this could potentially just be an integration test using rostest

Reply:
I moved this to the tools directory since it is a tool and not a localization requirement.  I use the non nodelet version in the offline bag testing, and the nodelet when replaying bags.  Testing with bags involves playing the bag, running the localizer in real time, and saving output to a results bag file.  Having the imu bias tester nodelet enables me run this along with the localizer so imu bias estimated poses are also saved to the results bag and can then be plotted using the plotting tool.